kbase.yml

```diff
@@ -11,7 +11,7 @@ service-language:
     python
 
 module-version:
-    2.3.1
+    2.4.0
 
 owners:
-    [msneddon, rsutormin, dylan, mclark58]
+    [dylan, jayrbolton, bsadkhin]
```

Comment:
should we be deleting old owners?

Reply:
I'm not really sure why we would keep perms for people not still on the project? I'm assuming they're not going to come back and work on this :P

Reply:
(I secretly hope they will!) Mostly I thought for giving them credit reasons.

Reply:
Oh good point, i wonder what would be the best way to credit authors besides this `owners` field? Display.yaml?